tests/python/contrib/test_hexagon/test_models.py

```diff
@@ -30,7 +30,7 @@ def get_mobilenet():
     """Download and import mobilenet model with ONNX"""
     onnx = pytest.importorskip("onnx")
 
-    model_url = "https://github.com/onnx/models/raw/main/vision/classification/mobilenet/model/mobilenetv2-7.onnx"  # pylint: disable=line-too-long
+    model_url = "https://github.com/onnx/models/raw/main/validated/vision/classification/mobilenet/model/mobilenetv2-7.onnx"  # pylint: disable=line-too-long
```

Comment:
seems the `https://github.com/apache/tvm/blob/main/tests/scripts/request_hook/request_hook.py` should be updated.

Reply:
I think it can work now